<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadBackgroundSkinDialog</class>
 <widget class="QWidget" name="DownloadBackgroundSkinDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>皮肤与窗口调整</string>
  </property>
  <widget class="DownloadTransitionAnimationLabel" name="background">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>442</width>
     <height>335</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>421</x>
     <y>6</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>442</width>
     <height>335</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#backgroundMask{background:rgba(255, 255, 255, 240);}</string>
   </property>
   <widget class="DownloadAnimationStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>40</y>
      <width>428</width>
      <height>250</height>
     </rect>
    </property>
    <widget class="QScrollArea" name="recommendScrollArea"/>
    <widget class="QScrollArea" name="userScrollArea"/>
    <widget class="QScrollArea" name="remoteScrollArea"/>
   </widget>
   <widget class="QLabel" name="skinTransparentLabel">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>300</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>皮肤透明</string>
    </property>
   </widget>
   <widget class="QPushButton" name="paletteButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>300</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>调色板</string>
    </property>
   </widget>
   <widget class="DownloadBackgroundPopWidget" name="skinTransparentButton" native="true">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>300</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="customSkin">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>300</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>本地皮肤</string>
    </property>
   </widget>
  </widget>
  <widget class="DownloadSkinAnimationWidget" name="skinAnimationSiwidget" native="true">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>10</y>
     <width>442</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>backgroundMask</zorder>
  <zorder>skinAnimationSiwidget</zorder>
  <zorder>topTitleCloseButton</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DownloadTransitionAnimationLabel</class>
   <extends>QLabel</extends>
   <header>downloadtransitionanimationlabel.h</header>
  </customwidget>
  <customwidget>
   <class>DownloadAnimationStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>downloadanimationstackedwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DownloadSkinAnimationWidget</class>
   <extends>QWidget</extends>
   <header>downloadfuntionanimationwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DownloadBackgroundPopWidget</class>
   <extends>QWidget</extends>
   <header>downloadbackgroundpopwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
